data.sql

```diff
@@ -62,3 +62,46 @@ VALUES
    UPDATE animals
    SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
    WHERE name = 'Angemon'  OR name = 'Boarmon';
+
+/* Day 4. */
+
+INSERT INTO vets (id, name, age, date_of_graduation) 
+   VALUES 
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23');
```

Comment:
I think you forgot to add the data for specialization to your data.sql file and instead you have data for vets twice. It happens to the best of us😅 kindly add the specialization data to fulfill the requirements of this milestone😉
![Screenshot_20230520_175024](https://github.com/Atril33/Vet_clinic/assets/44624138/79e17566-9821-469f-8342-837319bb0dea)